package.json

```diff
@@ -75,7 +69,25 @@
     "start": "webpack-dev-server",
     "build": "webpack -p",
     "build:analyze": "webpack -p --env.analyze",
-    "test": "karma start",
+    "test": "jest",
     "lint": "eslint 'src/**/*.js'"
+  },
+  "jest": {
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "src/**/*,js"
+    ],
+    "clearMocks": true,
+    "setupFilesAfterEnv": [
+      "<rootDir>/jest/setup.js"
+    ],
+    "modulePaths": [
+      "<rootDir>/src"
+    ],
+    "transform": {
+      "^.+\\.js?$": "babel-jest",
+      "^.+\\.html$": "<rootDir>/jest/htmlTransform.js"
+    },
+    "verbose": true
```

Comment:
Were you intending to keep this? Seems like it logs out successful tests also...

Reply:
I liked the output it produced. `verbose: false` was harder in travis to see exactly what test in a file was failing.